src/main/java/codeu/model/store/basic/SettingStore.java

```diff
@@ -0,0 +1,63 @@
+package codeu.model.store.basic;
+import codeu.model.data.Setting;
+
+import java.util.ArrayList;
+import java.util.List;
+import codeu.model.store.persistence.PersistentStorageAgent;
+
+public class SettingStore {
+    /**
+     * Singleton instance of UserStore.
+     */
+    private static SettingStore instance;
+
+    /**
+     * Returns the singleton instance of UserStore that should be shared between all servlet classes.
+     * Do not call this function from a test; use getTestInstance() instead.
+     */
+    public static SettingStore getInstance() {
+        if (instance == null) {
+            instance = new SettingStore(PersistentStorageAgent.getInstance());
+        }
+        return instance;
+    }
+
+    /**
+     * The PersistentStorageAgent responsible for loading Setting from and saving Users to Datastore.
+     */
+
+    private PersistentStorageAgent persistentStorageAgent;
+
+    //** The in list memory of the setting. */
+    private List<Setting > setting;
```

Comment:
Can this be a Map?

Reply:
on second thought i think this is fine as a list. but to tell settings apart i think we need to be a bit more sophisticated by added an enum so we can find settings in the list. Ill explain more in the settings class.